### PR TITLE
Refactor Escrow Finish Transactions

### PR DESCRIPTION
**Pull request by jonathanlei**
## High Level Overview of Change
PR split from https://github.com/ripple/explorer/pull/308

### Type of Change

- [X ] Refactor (non-breaking change that only restructures code)

### testing 
all tests passed

### CODE REVIEW

File: src/containers/shared/components/Transaction/EscrowFinish/Simple.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    fulfillment,
+  } = data.instructions
+
+  return (
+    <>
+      <SimpleRow label={t('finish_escrow')} data-test="escrow-finish">
+        <Account account={owner} />
+        {` - ${sequence}`}
+      </SimpleRow>
+      {condition && (
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/EscrowFinish/Simple.tsx:28]: condition, fulfillment, and destination need tests.  Can be future PR.

File: src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishDescription.test.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <I18nextProvider i18n={i18n}>
+      <BrowserRouter>
+        <Description data={tx} />
+      </BrowserRouter>
+    </I18nextProvider>,
+  )
+}
+
+describe('EscrowFinishDescription', () => {
+  it('renders description for EscrowFinish', () => {
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishDescription.test.tsx:20]: Needs test if the transaction fails.  Line 20 is not hit in TableDetail.tsx.

File: src/containers/shared/components/Transaction/EscrowFinish/test/mock_data/EscrowFinish.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (ckniffen) [inline src/containers/shared/components/Transaction/EscrowFinish/test/mock_data/EscrowFinish.json:1]: You should delete `src/containers/Transactions/test/mock_data/EscrowFinish.json`

**Review by ckniffen — APPROVED** [review]

File: src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishTableDetail.test.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { I18nextProvider } from 'react-i18next'
+import { BrowserRouter } from 'react-router-dom'
+import { mount } from 'enzyme'
+import { TableDetail } from '../TableDetail'
+import i18n from '../../../../../../i18nTestConfig'
+import summarize from '../../../../../../rippled/lib/txSummary'
+import mockEscrowFinish from './mock_data/EscrowFinish.json'
+
+const createWrapper = (tx: any) =>
```
Comment (JST5000) [inline src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishTableDetail.test.tsx:10]: Are any of these createWrapper functions equivalent across the various tests? (maybe it makes sense to have that be in a helper function?

File: src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishTableDetail.test.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { I18nextProvider } from 'react-i18next'
+import { BrowserRouter } from 'react-router-dom'
+import { mount } from 'enzyme'
+import { TableDetail } from '../TableDetail'
+import i18n from '../../../../../../i18nTestConfig'
+import summarize from '../../../../../../rippled/lib/txSummary'
```
Comment (JST5000) [inline src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishTableDetail.test.tsx:7]: Maybe a direct import would be better here?

File: src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishTableDetail.test.tsx (hunk 4 above)
Comment (ckniffen) [reply src/containers/shared/components/Transaction/EscrowFinish/test/EscrowFinishTableDetail.test.tsx:10]: Some are similar.  Its more just a naming convention.

**Review by pdp2121 — APPROVED** [review]

**Pull request merged**